src/jit/gentree.cpp

```diff
@@ -394,6 +408,88 @@ bool GenTree::IsNodeProperlySized() const
 }
 #endif
 
+/*****************************************************************************
+ *
+ *  When 'NODEBASH_STATS' is enabled in "jit.h" we record all instances of
+ *  an existing GenTree node having its operator changed. This can be useful
+ *  for two (related) things - to see what is being bashed (and what isn't),
+ *  and to verify that the existing choices for what nodes are marked 'large'
+ *  are reasonable (to minimize "wasted" space).
+ *
+ *  And yes, the hash function / logic is simplistic, but it is conflict-free
```

Comment:
IMO we should at least file an issue to change this s.t. it uses one of the preexisting hash tables. I think any option will need some changes in order to be able to use `ICorJitHost;:allocateMemory` for allocation (rather than `Compiler::compGetMem`), but I think that's a better approach long term.